main.py

```diff
@@ -111,10 +151,46 @@ def callback(ch, method, properties, body, ctx):
         )
         return
 
-    # Get a pid from the PIDService
-    pid_service = PIDService(ctx)
-    pid = pid_service.get_pid()
-    log.info(f"PID received: {pid}")
+    or_id = get_from_event(event, "tenant")
+    org_service = OrganisationsService(ctx)
+    cp_name = org_service.get_organisation(or_id)["cp_name_mam"]
+
+    # Check if we are dealing with essence or collateral
+    bucket = get_from_event(event, "bucket")
+    if bucket == "mam-collaterals":
+        object_key = get_from_event(event, "object_key")
+        collateral_type = object_key.split("/")[0]
+        media_id = object_key.split("/")[1]
```

Comment:
We know this is a weak point for now, since it explicitly depends on this line:
```
        f"s3://mam-collaterals/{event.destination_path_type()}/{event.media_id}/"
```
(https://github.com/viaacode/vrt-events-subtitle-available/blob/0d248a536341fa9aa8a83869e942b07e48b9545b/app/helpers/xml_helper.py#L22)
in the v-e-s-a.